openvdb_houdini/houdini/SOP_NodeVDB.cc

```diff
@@ -603,7 +603,7 @@ namespace {
 class DefaultOpenVDBOpPolicy: public houdini_utils::OpPolicy
 {
 public:
-    std::string getName(const houdini_utils::OpFactory&, const std::string& english) override
+    static std::string getName(const std::string& english)
```

Comment:
What's the motivation for splitting this out into a static member function?

Reply:
Primarily because ASWFOpenVDBOpPolicy calls SESIOpenVDBOpPolicy::getName() and SESIOpenVDBOpPolicy is not a base class of ASWFOpenVDBOpPolicy.

I thought it was clearer to adopt the same structure for all policy classes with regard to implementing getName() functionality rather than having some be static methods and some be instance methods, or alternatively having ASWFOpenVDBOpPolicy create an instance of SESIOpenVDBOpPolicy in order to call this method.

Reply:
I suggest moving the static `SESIOpenVDBOpPolicy::getName` into `DefaultOpenVDBOpPolicy` as a non-static member named `getLowercaseName`.

Reply:
Sure, that sounds reasonable to me.

Reply:
I've addressed your feedback now @pcucka - if you could have a quick look over this and check the changes make sense to you?